es6/context.js

```diff
@@ -64,7 +71,8 @@ export default {
       chalk.red(lineInfo.line.substr(lineInfo.column, length)) +
       lineInfo.line.substring(lineInfo.column + length, lineEnd);
     return {
-      info
+      info,
+      lineNumber: lineInfo.lineNumber
```

Comment:
You could do.

I guess long term it would be good if this returned context information without using `chalk`. When that happens, this could just return lineInfo..
Given that it might be best to either keep as-is for now, or set `info` on `lineInfo` and return that.